.ci/manywheel/build_cuda.sh

```diff
@@ -114,6 +114,16 @@ if [[ $USE_CUSPARSELT == "1" && $CUDA_VERSION == "11.8" ]]; then
         )
 fi
 
+# CUDA 11.8 have to ship the libnccl.so.2 with the binary
+if [[ $CUDA_VERSION == "11.8" ]]; then
```

Comment:
This is needed in order to package libnccl 2.25.1-1 into pytorch wheel for cuda 11.8